addons/binding/org.openhab.binding.milight/src/main/java/org/openhab/binding/milight/internal/protocol/MilightV3RGBW.java

```diff
@@ -0,0 +1,267 @@
+    }
+
+    @Override
+    public void setColorTemperature(int color_temp, MilightThingState state) {
+
+    }
+
+    @Override
+    public void setBrightness(int value, MilightThingState state) {
+        if (value <= 0) {
+            setPower(false, state);
+            return;
+        }
+
+        int newCommand = (int) Math.ceil((value * rgbwLevels) / 100.0) + 1;
+        setPower(true, state);
+
+        byte messageBytes[] = new byte[] { 0x4E, (byte) newCommand, 0x55 };
+        logger.debug("milight: send dimming packet '{}' to RGBW bulb channel '{}'", messageBytes, zone);
+        sendQueue.queue(messageBytes, uidc(type_offset, CAT_BRIGHTNESS_SET), true);
+
+        state.brightness = value;
+    }
+
+    @Override
+    public void changeBrightness(int relative_brightness, MilightThingState state) {
+        if (relative_brightness < 0) {
+            logger.debug("milight: sendDecrease");
+            byte messageBytes[] = null;
+
+            int newPercent = state.brightness - 10;
+            if (newPercent < 0) {
+                newPercent = 0;
+            }
+
+            if (state.brightness != -1 && newPercent == 0) {
+                setPower(false, state);
+            } else {
+                int decreasePercent = (int) Math.ceil((newPercent * rgbwLevels) / 100.0) + 1;
+                messageBytes = new byte[] { 0x4E, (byte) decreasePercent, 0x55 };
+                logger.debug("Bulb '{}' set to '{}' dimming Steps", zone, decreasePercent);
+                setPower(true, state);
+
+                sendQueue.queue(messageBytes, uidc(type_offset, CAT_BRIGHTNESS_SET), true);
+            }
+            state.brightness = newPercent;
+        } else if (relative_brightness > 1) {
+            logger.debug("milight: sendIncrease");
+            byte messageBytes[] = null;
+
+            int currentPercent = state.brightness;
+            int newPercent = currentPercent + 10;
+            if (newPercent > 100) {
+                newPercent = 100;
+            }
+
+            int increasePercent = (int) Math.ceil((newPercent * rgbwLevels) / 100.0) + 1;
+            messageBytes = new byte[] { 0x4E, (byte) increasePercent, 0x55 };
+            logger.debug("Bulb '{}' set to '{}' dimming Steps", zone, increasePercent);
+
+            setPower(true, state);
+
+            sendQueue.queue(messageBytes, uidc(type_offset, CAT_BRIGHTNESS_SET), true);
+            state.brightness = newPercent;
+        }
+    }
+
+    @Override
+    public void changeSpeed(int relative_speed, MilightThingState state) {
+        if (relative_speed > 0) {
+            logger.debug("milight: sendIncreaseSpeed");
+            byte messageBytes[] = null;
+
+            // message increaseSpeed rgb-w bulbs
+            messageBytes = new byte[] { 0x44, 0x00, 0x55 };
+
+            setPower(true, state);
+
+            sendQueue.queue(messageBytes, uidc(type_offset, CAT_SPEED_CHANGE), false);
```

Comment:
i agree. i had a remote in mind where the user presses the speed change button rapidly but gets actual changes just every 100ms and the commands queue up and in the end the user is faced with a mich greater change than expected